obspy/io/sac/tests/test_core.py

```diff
@@ -316,7 +316,7 @@ def test_undefinedB(self):
             self.assertEqual(tr2.stats.starttime.timestamp, 269596810.0)
```

Comment:
Thanks, Claudio.  I've made the change these changes in commit 39ccc896c047f824977cec242098aa24e34ee982 .  "B" is always written (because a Trace always knows its first sample time), and the reference time, if set, is not changed.  The current `test_undefinedB` now reflects these expectations.